tests/Forms/Forms.onSuccess.phpt

```diff
@@ -34,8 +33,23 @@ $form->onSuccess[] = function() use (& $called) {
 $form->onError[] = function() use (& $called) {
 	$called[] = 'err';
 };
+$form->fireEvents();
+Assert::same(array(1, 2, 'err'), $called);
 
 
+$called = array();
+$form = new Form;
+$form->addText('name');
+$form->addSubmit('submit');
+$form->onSuccess[] = function() use (& $called) {
```

Comment:
nn, ono se to nevolalo pokud error nastal v posledním onSuccess handleru - jejich celkový počet nemá vliv